lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -138,14 +220,15 @@ void PopulateSelection(
         return;
     }
 
-    SdfPath usdPath = usdItem->prim().GetPath();
+    SdfPath   usdPath = usdItem->prim().GetPath();
+    const int instanceIndex = usdItem->instanceIndex();
```

Comment:
Having the instanceIndex declared here might cause unused variable warning on older USD Imaging API version. Please move it down into the relevant conditional compilation block.